packages/orama/src/components/internal-document-id-store.ts

```diff
@@ -0,0 +1,70 @@
+import { Orama } from '../types.js';
+
+export type DocumentID = string | number
```

Comment:
Why allow both?

Reply:
I'm ok to introduce a little breaking change with this. Regarding the naming. Is this a documented or IndexId?

Reply:
This will bump Orama to `v1.1.0`

Reply:
Ok, so I will change all the references for `DocumentID` to `InternalID`, and I will only use `DocumentID` on `getByID` and when we return the documents from `search`.

> Regarding the naming. Is this a documented or IndexId?

I didn't understand your question but `DocumentID` is just an alias to reference the ID of the Document that was generated or was passed by the user, we you see this type, is referring to these two cases.

But I think we should add some documentation about it on Orama Docs, just to be clear about how we store IDs to be more efficient.

Reply:
Ok, understood you point